### PR TITLE
MOC 9 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/mistmeadow_vanisher.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Mistmeadow Vanisher
+ManaCost:2 WU
+Types:Creature Kithkin Wizard
+PT:3/2
+T:Mode$ Taps | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME becomes tapped, exile up to one target nonland, nontoken permanent. Return that card to the battlefield under its owner's control at the beginning of the next end step.
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Permanent.nonLand.nonToken | TargetMin$ 0 | TargetMax$ 1 | Mandatory$ True | TgtPrompt$ Select up to one target nonland, nontoken permanent | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DelTrig
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mistmeadow_vanisher.txt:6]: Valid syntax error

File: forge-gui/res/cardsfolder/upcoming/vulpine_harvester.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Vulpine Harvester
+ManaCost:3 W
+Types:Creature Phyrexian Fox
+PT:3/3
+T:Mode$ AttackersDeclared | ValidAttackers$ Phyrexian.YouCtrl | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Phyrexians you control attack, return target artifact card from your graveyard to the battlefield if its mana value is less than or equal to their total power.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Artifact.YouCtrl+cmcLEX | TgtPrompt$ Select target artifact in your graveyard | Origin$ Graveyard | Destination$ Battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vulpine_harvester.txt:6]: image

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vulpine_harvester.txt:6]: So a ChooseCard variable with a ConditionDefined$ Targeted or is there another way ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/vulpine_harvester.txt:6]: Try `ValidTgts$ Artifact.YouCtrl | ConditionDefined$ Targeted | ConditionPresent$ Card.cmcLEX`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**